src/App.js

```diff
@@ -102,6 +102,21 @@ function App(props) {
 
   if (fetchingUser) return <CircularProgress />;
 
+  const handleLogout = () => {
+    axios
+      .get(`${config.API_URL}/api/auth/logout`, { withCredentials: true })
+      .then(() => {
+        this.setState({
```

Comment:
we don't have this in functions (it's a classes thing). Does it work on your mashine?